### ISSUE TITLE
@types/styled-components Cannot redeclare block-scoped variable 'console'.

### ISSUE THREAD

**Issue by rami-res** [DefinitelyTyped/DefinitelyTyped#33275]
- [x] I tried using the `@types/styled-components` package and had problems because since of v.4.1.9 another conflicted dependencie was added (@types/react-native) and conflicts with @types/node .

![image](https://user-images.githubusercontent.com/8643060/53185025-d6070a00-3606-11e9-9fc9-1e61afc1f2cf.png)

- [x] I tried using the latest (3.3.3) stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I tried using "types" section in tsconfig.json file
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Is it right way to use "skipLibCheck": true in tsconfig.json file).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors so they can respond.
  - Authors of @types/styled-components: @Igorbek @Igmat @lavoaster @Jessidhia @jkillian 
  - Authors of @types/react-native: @alloy @huhuanming @iRoachie @skn0tt @timwangdev @kamal @nelyousfi @alexdunne @swissmanu @bm-software @tkrotoff @a-tarasyuk @mvdam @esemesek @mrnickel @souvik-ghosh @nossbigg

### how to reproduce:

- Fresh install react app by command
`yarn create react-app my-app-ts --scripts-version=react-scripts-ts`

- add styled components
`yarn add styled-components`
`yarn add -D @types/styled-components`

- import ThemeProvider to src/index.tsx and wrap <App/> to <ThemeProvider/>
```
import * as React from 'react';
import * as ReactDOM from 'react-dom';
import {ThemeProvider} from "styled-components";
import App from './App';
import './index.css';
import registerServiceWorker from './registerServiceWorker';

ReactDOM.render(
    <ThemeProvider theme={{}}>
        <App />
    </ThemeProvider>,
  document.getElementById('root') as HTMLElement
);
registerServiceWorker();
```
- run build command:
`yarn start`

- Expected result:
See the react app

- Current result:

my-app-ts/node_modules/@types/node/globals.d.ts
(164,13): Cannot redeclare block-scoped variable 'console'.
![image](https://user-images.githubusercontent.com/8643060/53184422-a572a080-3605-11e9-95b0-69207b98d7d5.png)

**Comment by Esemesek** [comment]
I had similar issues using both `react-native` and `styled-components`. I know that you can suppress it by providing the additional flag to Typescript compiler `--skipLibCheck`. You can find more information about it [here](https://www.typescriptlang.org/docs/handbook/compiler-options.html). It is definitely a hack to get things working temporarily.

**Comment by mikedloss** [comment]
Also getting this error (and those mentioned by @Esemesek), coming @types/rebass, issue is #33204

**Comment by rami-res** [comment]
@Esemesek I realized that it helps to fix @types/styled-components to  4.1.8 version
![image](https://user-images.githubusercontent.com/8643060/53228877-448dab80-368b-11e9-94fc-3b6d57ba15f4.png)

**Comment by alloy** [comment]
Fixed by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33302
